tests/tests.rs

```diff
@@ -31,20 +31,27 @@ fn compare_generated_header(header: &PathBuf,
     if !expected.is_file() {
         let file_name = expected.file_name().unwrap().to_owned();
         expected.pop();
-
-        if cfg!(feature = "testing_only_libclang_4") {
-            expected.push("libclang-4");
-        } else if cfg!(feature = "testing_only_libclang_3_9") {
-            expected.push("libclang-3.9");
-        } else if cfg!(feature = "testing_only_libclang_3_8") {
-            expected.push("libclang-3.8");
+        
+        let version = clang_version();
+        match version.parsed {
```

Comment:
We shouldn't panic on failure to parse the libclang version. This automatic-detection-of-libclang-version feature should only be an added convenience, not required. If we can't parse the libclang version (which happens because of crazy apple LLVM and things), then we should maintain the behavior we have now when no `testing_only_libclang_X` feature is used: complain that we aren't sure which expectation file to use and/or that the expectation file is missing.

I think maintaining this property is pretty important so that we don't *depend* on this "magic" behavior, just benefit when we can.